apps/mobile-wallet/src/components/AddressesTokensList.tsx

```diff
@@ -79,35 +78,31 @@ const AddressesTokensList = ({ addressHash, isRefreshing, parentModalId }: Addre
       </EmptyPlaceholder>
     )
 
+  const LoadingSection = (props: ComponentProps<typeof View>) => (
+    <LoadingSectionContainer {...props}>
+      <ActivityIndicator size={72} color={theme.font.primary} />
```

Comment:
So we are displaying a loader even though we are not loading any data, just because we are waiting for the animation to finish. Not sure I like it :/

Reply:
Personal opinion: 
- It's much better than the UI being glitchy and laggy
- You're brain cannot parse the list of token while it's moving, during in the ~1 second it takes for the modal to open 

Instead of a spinner, we could also have a placeholder (grey boxes likes Skeleton loaders, but not animated. Tried that, but it was laggy) and fade to the actual content once the animation is done. But it takes time to do it properly, I preferred to go for the fastest implementation.

Reply:
After testing this again, it seems like the "spinner time" could be reduced. Animation seems finished, however the spinner stays for a lil' while.

Reply:
yes, that's what I experienced as well